stories/constants/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { withDocs } from 'storybook-readme';
+import { storiesOf } from '@storybook/react';
+
+import * as COLORS from 'src/constants/colors';
+import * as CONSTANTS from 'src/constants';
+import { Typography } from 'src/shared-components';
+import ConstantsReadme from 'docs/constants.md';
+
+import renderConstantsMap from './renderConstantsMap';
+import Color from './Color';
+
+const stories = storiesOf('CONSTANTS', module);
+
+stories.add(
+  'available constants',
+  withDocs(ConstantsReadme, () => (
+    <div css="text-align: left;">
+      {
+        Object.keys(CONSTANTS).map(category => {
+          if (['COLORS'].includes(category)) {
```

Comment:
not really necessary for this since there is only one, but you could pull out in separate const so its easy to add to. 

```
const CONSTANT_BLACKLIST = ['COLORS'];
```
could also use a `filter` which might make it more readable, this way is more optimized tho